Source/monster.cpp

```diff
@@ -1850,33 +1850,34 @@ void AiAvoidance(Monster &monster)
 MissileID GetMissileType(MonsterAIID ai)
 {
 	switch (ai) {
-	case MonsterAIID::GoatMelee:
+	using enum MonsterAIID;

```

Comment:
This is C++20 only, but we need to support some targets that barely C++17

Reply:
... but looks like it did work, so feel fee to keep it like this.

Reply:
Didn't compile